build.gradle

```diff
@@ -501,7 +504,7 @@ project(':tools') {
     dependencies {
         compile project(':clients')
         compile 'net.sourceforge.argparse4j:argparse4j:0.5.0'
-        compile 'com.fasterxml.jackson.core:jackson-databind:2.5.4'
+        compile 'com.fasterxml.jackson.core:jackson-databind:' + jackson_version
```

Comment:
This might also work:
    compile 'com.fasterxml.jackson.core:jackson-databind:$jackson_version'